src/main/webapp/styles/addrecs.css

```diff
@@ -0,0 +1,88 @@
+/* --- Formatting for Popup Box --- */
```

Comment:
nit: it's clearer to use `recommendation` instead of its abbreviation, also please separate the two words in the file name with a dash

Reply:
Okay sounds good, thank you!